src/pages/index.js

```diff
@@ -0,0 +1,181 @@
+import "../pages/index.css";
+import Card from "../components/Card.js";
+import FormValidator from "../components/FormValidator.js";
+import PopupWithForm from "../components/PopupWithForm.js";
+import PopupWithImage from "../components/PopupWithImage.js";
+import Section from "../components/Section.js";
+import UserInfo from "../components/UserInfo.js";
+import { initialCards, validationSettings } from "../utils/constants.js";
+import { api } from "../components/Api.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
+
+// DOM Elements
+const profileEditButton = document.querySelector("#profile__edit-button");
+const profileAddButton = document.querySelector(".profile__add-button");
+const profileTitleInput = document.querySelector("#profile-title-input");
+const profileDescriptionInput = document.querySelector("#profile-description-input");
+const profileEditForm = document.querySelector("#profile-edit-modal .modal__form");
+const profileAddForm = document.querySelector("#profile-add-modal .modal__form");
+const avatarEditButton = document.querySelector("#profile-avatar-edit");
+
+// User Info
+const userInfo = new UserInfo({
+  nameSelector: "#profile__title",
+  jobSelector: "#profile__description",
+  avatarSelector: "#profile-avatar",
+});
+
+// Popups
+const imagePopup = new PopupWithImage("#image__open-modal");
+imagePopup.setEventListeners();
+
+const confirmationPopup = new PopupWithConfirm({
+  popupSelector: "#delete-card-modal",
+});
+
+confirmationPopup.setEventListeners();
+const profileEditPopup = new PopupWithForm("#profile-edit-modal", handleProfileEditSubmit);
+profileEditPopup.setEventListeners();
+
+const profileAddPopup = new PopupWithForm("#profile-add-modal", handleCardAddSubmit);
+profileAddPopup.setEventListeners();
+
+const avatarEditPopup = new PopupWithForm("#avatar-edit-modal", handleAvatarEditSubmit);
+avatarEditPopup.setEventListeners();
+
+// Form Validators
+const editFormValidator = new FormValidator(validationSettings, profileEditForm);
+editFormValidator.enableValidation();
+
+const addFormValidator = new FormValidator(validationSettings, profileAddForm);
+addFormValidator.enableValidation();
```

Comment:
* Validation of the `avatar` form doesn't work. Please, create a validator for the `avatar` form